Dockerfile

```diff
@@ -48,6 +48,10 @@ RUN apk add --no-cache --virtual .crate-rundeps \
     && ln -s /usr/bin/python3 /usr/bin/python \
     && apk del .build-deps
 
+# install crash
+RUN curl -o /usr/local/bin/crash https://cdn.crate.io/downloads/releases/crash_standalone_latest \
```

Comment:
I am not sure about it. Would it make sense to set the version explicitly?

Reply:
We could use [arg](https://docs.docker.com/engine/reference/builder/#arg) to make to settable at build time for reproducable builds. And yep, I'd set it to an explicit version. (for the same reason, reproducable builds)

Reply:
I wanted to use `ARG`, but we don't build the image with the `docker build` command where we could pass the version as an argument, but rather we tag the commit with changes in the  `Dockerfile` file and pass the tag to `hub.docker.net` to build the image. Therefore, I just used `ENV`, as it is done in the crate layer.